sig-node/annual-report-2024.md

```diff
@@ -4,40 +4,39 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+SIG-Node remains a structural piece of the Kubernetes community and the span of the work done in 2024 highlights that.
+As the community continues rallying behind AI use cases and identifying gaps with Kubernetes as a platform for LLM training and serving,
+SIG-Node made strides in multiple AI related areas. DRA structured parameters made it to beta, meaning more flexible scheduling and allocation of
+device resources is now possible. In 2025 there will be a lot of continued work on DRA, including enhancing drivers to be able to report device health
+and Kubernetes components be able to react to that, extending DRA to support advanced networking use cases, device taints and tolerations, and lots more!
+Outside of DRA, OCI image volume mounts have been added as alpha in 2024, allowing users to mount AI models into containers via a separate image (and one day artifact) instead
+of a model car or embedding it in the container image. Also, work like in-place pod resize and pod level resource limits will unlock use cases for power AI users: allowing more flexibility
+in pod resource limit calculation at both initialization and during runtime.
+
+Plenty of work has been being done outside of AI as well! SIG-Node remains the top SIG in KEPs progressing, moving forward on 13, 16, and 17 KEPs between 1.30, 1.31, and 1.32 respectively.
+Lots of progress has been made in the CPU manager: like adding support for split uncore cache, adding a policy option for restricting resrevedSystemCPUS and a new static policy for optimizing CPU alignment.
+We have also worked on some long awaited linux technologies like user namespaces, swap, AppArmor, ephemeral storage quotas, recursive read only mounts, and better support for supplemental groups,
+as well as announced feature freeze on cgroupv1.
+
+All of these features don't even begin to cover the amount of CI stabilization, bug fixes, and other work the SIG is doing. We remain a productive (albeit, occasionally overbooked) SIG. To help keep up with all
+of the work, we've inducted one new approver Sergey Kanzhelev, reinducted a formerly emertius approver Tim Allclair, welcomed a new SIG chair Peter Hunt, as well as began crafting a role to help KEP authors follow along the KEP process, currently called the KEP wranglers.
```

Comment:
Do we have a summary about subproject/sub component approver addition last year?
- I find https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/OWNERS and https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/OWNERS only ffromani was added in early 2025. The previous adding is in 2022.

Reply:
we didn't make meaningful progress on this